test/client/fixtures/sandbox/node/classes-test.js

```diff
@@ -56,6 +56,26 @@ if (window.EventSource) {
 
         ok(eventSource instanceof EventSource);
     });
+
+    test('should have static constants (GH-1106)', function () {
+        ok(EventSource.hasOwnProperty('CONNECTING'));
+        ok(EventSource.hasOwnProperty('OPEN'));
+        ok(EventSource.hasOwnProperty('CLOSED'));
+    });
+
+    test('should process url to proxy with special flag (GH-1106)', function () {
```

Comment:
Add tests for the cases when EventSource is closed:
-  from client - `eventSource.close()`
- from server - status code does not equal `200`